packages/suite-web-landing/pages/start.tsx

```diff
@@ -1,137 +0,0 @@
-import React, { useState } from 'react';
```

Comment:
We still need /start page, it's a landing page for every new owner of Trezor device... It looks similar to index, but translation keys (and texts) are different.

Reply:
14a42c2